docker-bake.hcl

```diff
@@ -40,10 +45,8 @@ variable "TARGETS" {
 function "tags" {
   params = [image]
   result = [
-    "${REGISTRY}${ORGANIZATION}/${image}:${VERSION}",
-    "${REGISTRY}${ORGANIZATION}/${image}:python-${PYTHON_VERSION}",
```

Comment:
@unkcpz why was this removed? I think we want to keep using these tags on Dockerhub, no? Also, what is the purpose of the `newly-build` tag?

btw: Thanks for working on this! Looks much cleaner now.

Reply:
This is not used for push to the remote registry. When push to the remote, tags are defined from action it self, see [here](https://github.com/aiidalab/aiidalab-docker-stack/blob/5ed9b82f11ee02c1dac85d0afc00d810eba45e59/.github/workflows/docker-push.yml#L59-L66)

> Also, what is the purpose of the newly-build tag?

In the merge architecture action, the images are downloaded from artifacts uploaded. The newly-build tag is used to point to the image built from the github workflow. It can be `latest`, but if the latest is not found from the downloaded artifact, there are also remote `latest` tag that will be pulled automatically from the registry, which may cause the issue that the test may not run on the image that builds from the PR.

Reply:
Thanks for the explanation. How are the latest and new-build tags distinguished when you have multiple PRs?

Reply:
If you mean the images build in one PR, if a new commit is pushed to remote, I use concurrency workflow to make sure only one build is running https://github.com/aiidalab/aiidalab-docker-stack/blob/c97288e0c7b98cd6c52002f56aca6cc6a7e14310/.github/workflows/docker.yml#L55-L59

Thus the `pr-xxx` and `sha-xxx` tag will be generated only from one specific commit.